src/components/EditGrower.js

```diff
@@ -161,6 +161,12 @@ const EditGrower = (props) => {
               ))}
             </Grid>
           ))}
+          <TextField
+            className={classes.textInput}
+            label="Grower-entered organization"
+            value={getValue('organization')}
+            disabled
+          ></TextField>
```

Comment:
For better readability and more concise code, you can create the TextField component like this:
```javascript
          <TextField
            className={classes.textInput}
            label="Grower-entered organization"
            value={getValue('organization')}
            disabled
          />
```

Reply:
Agreed, I completely overlooked that.

Reply:
Updated